resources/tutorial/lapnig_fixed.asciipb

```diff
@@ -0,0 +1,8 @@
+fixed_values {
+  mean: 20
+  var: 10
+  shape: 2
+  scale: 1
+  mh_mean_var: 60;

```

Comment:
is this a reasonable value? I would have expected values around 0.5 for the mean and 0.25 for the log scale to give reasonable results. What is the acceptance rate now?

Reply:
We tested the values you're proposing and we obtained acceptance rate of ~0.54. 
With the following values we obtained 0.256359
{ mean: 20
  var: 10
  shape: 2
  scale: 1
  mh_mean_var: 10;
  mh_log_scale_var: 1.3; }

Reply:
btw this was on the galaxy dataset :)

Reply:
We changed the values in the latest commit tuned for the tutorial dataset. We obtained 0.225 acceptance rate with the following values
{
  mean: 0
  var: 10
  shape: 2
  scale: 1
  mh_mean_var: 10;
  mh_log_scale_var: 1;
}